integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/AppReceiptPutMessageStrategy.java

```diff
@@ -36,16 +38,19 @@
         }
     }
 
-    public AppReceiptMessageStrategy(MessageSender messageSender) {
+    public AppReceiptMessageStrategy(MessageSender messageSender, IntegrasjonspunktProperties properties) {
         this.messageSender = messageSender;
+        this.properties = properties;
     }
 
     @Override
     public PutMessageResponseType send(EDUCore request) {
         Audit.info("Received AppReceipt", markerFrom(request));
         AppReceiptType receipt = request.getPayloadAsAppreceiptType();
         if (asList("OK", "WARNING", "ERROR").contains(receipt.getType())) {
-            request.swapSenderAndReceiver();
+            if (!properties.getNoarkSystem().getType().toLowerCase().equals("ephorte")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "ephorte" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1132)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1157)